api/types/constants.go

```diff
@@ -413,6 +413,14 @@ const (
 	// found via automatic discovery, to avoid re-running installation
 	// commands on the node.
 	AWSInstanceIDLabel = TeleportNamespace + "/instance-id"
+	// AzureSubscriptionIDLabel is used to identify virtual machines by Azure
+	// subscription ID found via automatic discovery, to avoid re-running
+	// installation commands on the node.
+	AzureSubscriptionIDLabel = TeleportNamespace + "/subscription-id"
+	// AzureVMIDLabel is used to identify virtual machines by ID found
+	// via automatic discovery, to avoid re-running installation commands
+	// on the node.
+	AzureVMIDLabel = TeleportNamespace + "/vm-id"
```

Comment:
Should we write the strings to be more Azure-specific, or name the constants in a more generic manner?

Eg, "/azure-subscription-id" and "/azure-vm-id" ?